src/pages/patientView/mutation/column/AlleleFreqColumnFormatter.tsx

```diff
@@ -31,12 +32,30 @@ export default class AlleleFreqColumnFormatter {
 
             const bar = (<rect x={barX[mutation.sampleId]} y={barY} width={AlleleFreqColumnFormatter.barWidth} height={barHeight} fill={sampleManager.getColorForSample(mutation.sampleId)}/>);
 
-            const circleRadius = 6;
             const sampleId = mutation.sampleId;
-            const component = sampleManager.getComponentForSample(sampleId);
+            const isUncalled = mutation.geneticProfileId.endsWith(GENETIC_PROFILE_UNCALLED_MUTATIONS_SUFFIX);
+            let opacity: number = 1;
+            let extraTooltipText: string = '';
+            if (isUncalled) {
+                if (altReads > 0) {
+                    opacity = 0.1;
+                    extraTooltipText = "Mutation has supporting reads, but wasn't called";
```

Comment:
we should write some tests for this code.  let me know if you want some help with that.